Resources/views/widgets/flight_board.blade.php

```diff
@@ -49,9 +49,13 @@
               </a>
             </td>
             <td>
+              @if($flight->aircraft_id === null)
+                Not Assigned
```

Comment:
No need to print a text and use `@else` ... Just check for the field with `filled()` to get rid of the exception, all will be ok.